037-WindowsVirtualDesktop/Coach/03-Implement-Manage-Storage.md

```diff
@@ -0,0 +1,53 @@
+# Challenge 3: Implement and Manage your storage for FsLogix
+
+[< Previous Challenge](./02-Implement-Manage-Network.md) - **[Home](./README.md)** - [Next Challenge >](./04-Create-Manage-Images.md)
+
+## Notes & Guidance
+
+1. Capacity planning
```

Comment:
I would use a bullet here, not a number. You don't have a #2 so it's not really an ordered list.